packages/utilities/fast-colors/src/color-converters.spec.ts

```diff
@@ -106,10 +106,11 @@ describe("Color converter functions", () => {
                 data.rgba.a
             );
 
-            const hsl: ColorHSL = rgbToHSL(rgb);
-            expect(hsl.h).to.be.closeTo(data.hsl.h, testPrecision);
-            expect(hsl.s).to.be.closeTo(data.hsl.s, testPrecision);
-            expect(hsl.l).to.be.closeTo(data.hsl.l, testPrecision);
+            const hsl: ColorHSL | null = rgbToHSL(rgb);
+            expect(hsl).not.to.be.null;
+            expect(hsl!.h).to.be.closeTo(data.hsl.h, testPrecision);
```

Comment:
are we handling null in rgbToHSL?

Reply:
Yes, If you check the definition of rgbToHSL it checks in the very first line whether the rgb parameter is of null type or of `ColorRGBA64`. If a null type is found it returns null else proceed with the functionality. I'm also checking the return type of `hsl` variable is not null (on line number 110).

Would you like if I write a test case to check if it returns null on passing null?